front_end/panels/ai_chat/LLM/LLMClient.ts

```
@@ -383,39 +414,91 @@ export class LLMClient {
     }
   }
 
+  /**
+   * Static method to fetch models from Cerebras API (for UI use without initialization)
+   */
+  static async fetchCerebrasModels(apiKey: string): Promise<any[]> {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)